<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TicTacToe</class>
 <widget class="QMainWindow" name="TicTacToe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>503</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TicTacToe</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qradialgradient(spread:repeat, cx:1, cy:1, radius:0.5, fx:1, fy:1, stop:0 rgba(255, 241, 71, 130), stop:0.1875 rgba(251, 255, 0, 145), stop:0.261364 rgba(255, 255, 0, 69), stop:0.57754 rgba(255, 203, 0, 130), stop:0.602273 rgba(247, 255, 0, 208), stop:0.755682 rgba(255, 218, 71, 130), stop:0.863636 rgba(249, 255, 36, 69), stop:0.926136 rgba(250, 255, 87, 69), stop:1 rgba(255, 255, 0, 69));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QPushButton" name="btn_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>50</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">/* From https://css.glass */
background: rgba(72, 124, 200, 0.33);
border-radius: 16px;
box-shadow: 0 4px 30px rgba(0, 0, 0, 0.1);
backdrop-filter: blur(8.8px);
-webkit-backdrop-filter: blur(8.8px);
border: 4px double rgba(200, 280, 200, 1);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="btn_6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>50</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">/* From https://css.glass */
background: rgba(72, 124, 200, 0.33);
border-radius: 16px;
box-shadow: 0 4px 30px rgba(0, 0, 0, 0.1);
backdrop-filter: blur(8.8px);
-webkit-backdrop-filter: blur(8.8px);
border: 4px double rgba(200, 280, 200, 1);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="btn_8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>50</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">/* From https://css.glass */
background: rgba(72, 124, 200, 0.33);
border-radius: 16px;
box-shadow: 0 4px 30px rgba(0, 0, 0, 0.1);
backdrop-filter: blur(8.8px);
-webkit-backdrop-filter: blur(8.8px);
border: 4px double rgba(200, 280, 200, 1);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="btn_5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>50</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">/* From https://css.glass */
background: rgba(72, 124, 200, 0.33);
border-radius: 16px;
box-shadow: 0 4px 30px rgba(0, 0, 0, 0.1);
backdrop-filter: blur(8.8px);
-webkit-backdrop-filter: blur(8.8px);
border: 4px double rgba(200, 280, 200, 1);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="btn_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>50</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">/* From https://css.glass */
background: rgba(72, 124, 200, 0.33);
border-radius: 16px;
box-shadow: 0 4px 30px rgba(0, 0, 0, 0.1);
backdrop-filter: blur(8.8px);
-webkit-backdrop-filter: blur(8.8px);
border: 4px double rgba(200, 280, 200, 1);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="btn_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>50</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">/* From https://css.glass */
background: rgba(72, 124, 200, 0.33);
border-radius: 16px;
box-shadow: 0 4px 30px rgba(0, 0, 0, 0.1);
backdrop-filter: blur(8.8px);
-webkit-backdrop-filter: blur(8.8px);
border: 4px double rgba(200, 280, 200, 1);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="btn_7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>50</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">/* From https://css.glass */
background: rgba(72, 124, 200, 0.33);
border-radius: 16px;
box-shadow: 0 4px 30px rgba(0, 0, 0, 0.1);
backdrop-filter: blur(8.8px);
-webkit-backdrop-filter: blur(8.8px);
border: 4px double rgba(200, 280, 200, 1);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="btn_9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>50</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">/* From https://css.glass */
background: rgba(72, 124, 200, 0.33);
border-radius: 16px;
box-shadow: 0 4px 30px rgba(0, 0, 0, 0.1);
backdrop-filter: blur(8.8px);
-webkit-backdrop-filter: blur(8.8px);
border: 4px double rgba(200, 280, 200, 1);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="btn_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>50</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">/* From https://css.glass */
background: rgba(72, 124, 200, 0.33);
border-radius: 16px;
box-shadow: 0 4px 30px rgba(0, 0, 0, 0.1);
backdrop-filter: blur(8.8px);
-webkit-backdrop-filter: blur(8.8px);
border: 4px double rgba(200, 280, 200, 1);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QRadioButton" name="cpu_radio_btn">
      <property name="font">
       <font>
        <family>Britannic Bold</family>
        <pointsize>16</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="text">
       <string>player vs cpu</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QRadioButton" name="player_radio_btn">
      <property name="font">
       <font>
        <family>Britannic Bold</family>
        <pointsize>16</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="text">
       <string>player vs player</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="new_game_btn">
      <property name="font">
       <font>
        <family>Modern No. 20</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(72, 124, 200, 0.33);
border-radius:10px;
box-shadow: 0 4px 30px rgba(0, 0, 0, 0.1);
backdrop-filter: blur(8.8px);
-webkit-backdrop-filter: blur(8.8px);
border: 2px solid rgba(200, 280, 200, 1);
color: rgb(0, 0, 255);</string>
      </property>
      <property name="text">
       <string>NewGame</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>534</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
